source/core/slang-performance-profiler.cpp

```diff
@@ -1,13 +1,9 @@
 #include "slang-performance-profiler.h"
 #include "slang-dictionary.h"
+#include <unordered_map>
```

Comment:
I guess we can just use our own containers for consistency with the rest of the code base.

Reply:
yea, I can change the std::vector to List then.

Reply:
Done